src/sentry/templates/sentry/emails/transactions.html

```diff
@@ -46,5 +46,33 @@
             {% endfor %}
         </tr>
         {% endif %}
+        {% if span_evidence_key_value %}
+        {% for item in span_evidence_key_value %}
+            {% if item.is_multi_value %}
```

Comment:
🤔 tricky. Might be cleaner to use a Django filter to wrap single strings in an array, so it's always an iterable, instead of having to manually declare this. That, or always returns an array from the Python part. NBD

Reply:
Yeah, its also tricky, because we might want an array to be the value and take up the whole role, we do this in the frontend all the time in span evidence, and I was trying to make this consistent to how span evidence works there.

It would look like this
![image](https://user-images.githubusercontent.com/44422760/216673796-e452a185-3b38-429d-b4d3-49a3a6052d80.png)

![image](https://user-images.githubusercontent.com/44422760/216673832-3d3a7ba3-2d78-48a9-a411-de2151e8ea82.png)